docs/dg/learningBasics.md

```diff
@@ -0,0 +1,416 @@
+<variable name="title">Learning the Basics</variable>
+<frontmatter>
+  title: "{{ title | safe }}"
+  pageNav: 3
+</frontmatter>
+
+{% from 'scripts/macros.njk' import embed, step with context %}
+
+<h1 class="display-4"><md>{{ title }}</md></h1>
+
+<div class="lead">
+
+This is a learning guide for developers who are new to RepoSense.
+</div>
+
+<box type="warning" seamless>
+
+Depending on what you have already known and what you would like to work on (backend or frontend), you may find certain sections irrelevant to you and you can skip them accordingly.
```

Comment:
```suggestion
Depending on what you know already and what you would like to work on (i.e., backend or frontend), you may find certain sections irrelevant to you and you can skip them accordingly.
```

Reply:
Thanks for the suggestion. I will change it accordingly